### PR TITLE
Add gulp support, remove old workflow bits

### PR DESCRIPTION
**Pull request by tamasd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,13 +185,3 @@ Open `docker-compose.override.yml` and follow the instructions in the `cli` and
 ## Running tests (optional)
 
 `docker-compose -f docker-compose.yml -f docker-compose.tests.yml -f docker-compose.override.yml run --rm tests`
-
-## Updating devportal themes and modules
```
Comment (mxr576) [inline README.md:183]: Please update the PR's title because it does not just add Gulp support.

Comment (mxr576) [reply README.md:183]: Or move this to a new PR. This closes #22 btw.

Comment (tamasd) [reply README.md:183]: done

**Review by lekob — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -163,6 +163,12 @@ While it is not strictly necessary to enable the redis module, it is recommended
 - `docker-compose down`
 
   Destroys the containers (permanently deletes the state).
+  
+## Gulp
+
+Copy `.env.js` as `env.js` into the project root from [the Gulp component](https://github.com/Pronovix/gulp), then run
+`docker-compose run --rm gulp sh -c 'npm run gulp'` to build the CSS, or
```
Comment (kittiradovics) [inline README.md]: Could you improve this part of the documentation? Something like:

`docker-compose run --rm gulp sh -c 'npm run gulp'` to build the CSS, or
`docker-compose run --rm gulp sh -c 'npm run gulp watch'` to continuously build the CSS when a change happens in `.scss` files.
You can use `--debug` to build the non-minified CSS with sourcemaps, e.g. `docker-compose run --rm gulp sh -c 'npm run gulp watch -- --debug'` (standalone `--` is needed to pass the argument to Gulp).

**Review by kittiradovics — CHANGES_REQUESTED** [review]

**Review by kittiradovics — APPROVED** [review]

**Pull request merged**